--- luminance-hdr-2.3.1/src/Common/TranslatorManager.cpp.orig	2013-04-07 20:41:14.000000000 +0200
+++ luminance-hdr-2.3.1/src/Common/TranslatorManager.cpp	2016-09-09 18:49:00.625142009 +0200
@@ -83,7 +83,7 @@
 
     if ( qtTranslator->load(QString("qt_") + lang, QString("i18n")) ||
          // qtTranslator->load(QString("qt_") + lang, I18NDIR) )
-         qtTranslator->load(QString("qt_") + lang, QLibraryInfo::location(
+         qtTranslator->load(QString("qt4_") + lang, QLibraryInfo::location(
                                 QLibraryInfo::TranslationsPath)) )
     {
         QCoreApplication::installTranslator(qtTranslator.data());
